guides/common/modules/con_ipv4-and-ipv6-requirements.adoc

```diff
@@ -0,0 +1,44 @@
+[id="ipv4-and-ipv6-requirements_{context}"]
+= IPv4 and IPv6 requirements
+
+You can install {Project} in an IPv4 network or in an IPv6 network.
+
+ifdef::satellite[]
+[NOTE]
+====
+Dual-stack {Project} installation that uses both IPv4 and IPv6 is not supported.
+====
+endif::[]
+
+The following requirements apply to installations in an IPv4 network:
+
+* Ensure that an IPv6 loopback is configured.
+* Do not disable IPv6 in kernel by adding the `ipv6.disable=1` kernel parameter.
+ifdef::satellite[]
++
+For a supported way to disable the IPv6 protocol, see link:https://access.redhat.com/solutions/5045841[How do I disable the IPv6 protocol on Red Hat Satellite and/or Red Hat Capsule server?] in Red{nbsp}Hat Knowledgebase.
+endif::[]
+
+The following requirements apply to installations in an IPv6 network:
+
+* You must deploy an external DHCP IPv6 server as a separate, unmanaged service to bootstrap clients into GRUB2.
+GRUB2 configures IPv6 networking by using DHCPv6 or by assigning a static IPv6 address.
```

Comment:
I think GRUB can also do SLAAC (stateless IPv6 address assignments). A problem is that we can't provide the required information where to boot from.

Taking a step back, there are 3 ways to configure IPv6 addresses:
* Static: user provides this info manually
* SLAAC: https://en.wikipedia.org/wiki/IPv6#Stateless_address_autoconfiguration_(SLAAC)
* DHCPv6

SLAAC can only provide a limited amount of information. It's sufficient to provide addresses and DNS, but in particular for provisioning, it can't provide info like the TFTP server address and filename or HTTPBoot URL which we need.

That information is crucial because on fresh installations you don't have anything. We download GRUB (or something else) from a TFTP or HTTPBoot location.